configure.ac

```diff
@@ -463,7 +462,9 @@ darwin*)
 esac
 
 dnl Eventually remove this
-CFLAGS="$CFLAGS -I${prefix}/include/heartbeat"
```

Comment:
@lge make a template and have autoconf fill them in:  https://github.com/ClusterLabs/pacemaker/blob/master/lib/pacemaker.pc.in